warehouse/templates/manage/account.html

```diff
@@ -437,7 +437,8 @@ <h2 class="sub-title">{% trans %}Change password{% endtrans %}</h2>
         </ul>
       </div>
       <div>
-        <input value="Update password" class="button button--primary" type="submit" data-target="password-match.submit">
+        {% set updateButtonText=gettext("Update password") %}
+        <input value="{{ updateButtonText }}" class="button button--primary" type="submit" data-target="password-match.submit">
```

Comment:
Just curious, why use `gettext` over `trans` here?

```suggestion
        <input value="{% trans %}Update Password{% endtrans %}" class="button button--primary" type="submit" data-target="password-match.submit">
```

Reply:
I have no idea :tired_face: :laughing:

Reply:
OK, fine with me either way I think. If you want to accept this suggestion you'll likely need to run `make translations` again though.